docker/kibana/kibana.yml

```diff
@@ -5,3 +5,4 @@ monitoring.ui.container.elasticsearch.enabled: true
 
 elastic.apm.active: true
 elastic.apm.serverUrl: "http://apm-server:8200"
+xpack.task_manager.monitored_stats_health_verbose_log.enabled: true
```

Comment:
is this option available for all the releases in the 7.x line? Just in case it could produce a side effect if using an obsolete version. Though, no idea if the `kibana.yml` supports options that are not available yet